src/assets/sitemapper.js

```diff
@@ -173,7 +173,7 @@ export default class Sitemapper {
   /**
    * Requests the URL and uses parseStringPromise to parse through and find the data
    *
-   * @private
+   * @public
```

Comment:
Does changing this comment change any functionality? Or is it more of a formal way of defining it public. I believe in JS the comment doesn't actually prevent anything from being accessed. 

I don't see any issues with defining it as public.